keras_cv/layers/object_detection/multi_class_non_max_suppression.py

```diff
@@ -107,7 +107,9 @@ def call(self, box_prediction, class_prediction):
             "num_detections": valid_det,
         }
         # this is required to comply with KerasCV bounding box format.
-        return bounding_box.mask_invalid_detections(bounding_boxes)
+        return bounding_box.mask_invalid_detections(
+            bounding_boxes, output_ragged=True
```

Comment:
Could we expose `output_ragged` as a param?

Reply:
Luke had previously done this, but I left a comment about it. (Sorry for the churn!)

Basically my take is that it's sorta a weird config option to have. In general, parameters that look like `output_xyz` should be instead handled as post-processing by the caller IMO.

Feel free to disagree!

Reply:
OK, fine by me